### DISCUSSION TITLE
The hardware on this system lacks support for NEON SIMD extensions. We now require NEON or equivalent architecture extensions on ARM-based machines.

### DISCUSSION THREAD

**Discussion by TheExpertNoob** [guysoft/FullPageOS#621] [Show and tell]
Not an issue per-se, but a fix I found for older hardware. Loaded up the latest Nightly (10-28-2024) on a Pi Zero W (2017) with the Raspberry Pi Imager and eventually got a pop-error mentioned in the title. Reverted back to an older version of chromium-browser with the commands mentioned below.
```
wget https://archive.raspberrypi.org/debian/pool/main/c/chromium-browser/chromium-codecs-ffmpeg-extra_104.0.5112.105-rpt2_armhf.deb

wget https://archive.raspberrypi.org/debian/pool/main/c/chromium-browser/chromium-browser_104.0.5112.105-rpt2_armhf.deb

sudo apt install ./chromium-codecs-ffmpeg-extra_104.0.5112.105-rpt2_armhf.deb

sudo apt install ./chromium-browser_104.0.5112.105-rpt2_armhf.deb

sudo reboot
```
Hopefully other people that come across this will find this helpful.

**Comment by marko-pi** [comment]
I have the same problem, but I get "Connection timed out." when I try to install an old version of Chromium.

By the way, I got this problem automatically after a regular Rasperry Pi OS update.

This is a real issue and should be resolved.

**Comment by guysoft** [comment]
Converting to a discussion

**Comment by rothos** [comment]
Thank you @TheExpertNoob !!! Fixed my issue.

**Comment by Luke-L** [comment]
Flashing Pi OS Buster fixed my issue. Obv there are security issues, so YMMV. https://downloads.raspberrypi.org/raspbian/images/raspbian-2020-02-14/
